src/pages/dataset-list-view/dataset-list-view.js

```diff
@@ -61,15 +61,6 @@ const DatasetListView = (props) => {
     return <div className="result-count">{`${resultCountText}${resultCountQueryText}`}</div>
   }
 
-  if (isError) {
-    return (
-      <ErrorComponent
-        errorText={
-          "We were unable to fetch the datasets, please refresh the page to try again"
-        }
-      />
-    )
-  } else {
```

Comment:
I don't think this should go away.  If our call `/search` fails, we still need to show this error.

Reply:
I couldn't find any reference of this being used - if the calls to the API fail they are caught by the network-loading-element and that element displays the error.

Reply:
Weird.  It looks like both this and the NetworkErrorComponent are wired up to use the `getDataSetError` selector.  If that's true, then we would technically be showing this component twice when the network call fails on the DatasetListView.  Good catch.